### PR TITLE
docs: update readme / reorganize

### PR DESCRIPTION
**Pull request by cdrage**
docs: update readme / reorganize

### What does this PR do?

Updates the README with the following:
* Removes references to OpenShift
* Updates the main screenshot
* Reorganizes / rewords some parts of the README
* Changes the link to download to the website instead of the GitHub
  release page

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

N/A

### How to test this PR?

View it :)

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# Podman Desktop
-
-Install, configure and keep container engine (Podman) up to date. System tray to keep it running and switch Kubernetes contexts. 
-Desktop App to interact with containers and pods running in Podman. Connect and deploy Pods to Kubernetes/OpenShift environments.
+Podman Desktop - A graphical tool for developing containers locally and on Kubernetes
```
Comment (slemeur) [inline README.md]: I don't think the tag line here is right. 

We previously define this one: "a GUI for Containers and Kubernetes for application developers". We can surely refine/iterate on it.

Do we need to repeat the name of the tool in the title? Should we go straight into something like "Containers and Kubernetes for application developers"

File: README.md [hunk 2]
```diff
@@ -15,70 +15,74 @@ Desktop App to interact with containers and pods running in Podman. Connect and
 - [**Code of Conduct**](#code-of-conduct)
 - [**License**](#license)
 
-## Join our Early Adopter Program!
+## Overview
```
Comment (slemeur) [inline README.md:18]: Maybe the title should be "What's Podman Desktop"

File: README.md [hunk 3]
```diff
@@ -15,70 +15,74 @@ Desktop App to interact with containers and pods running in Podman. Connect and
...
 
-We are seeking users who are interested in helping us improve Podman Desktop.
 
-Register your interest to join our Early Adopter Program by filling <a href="https://forms.gle/ow73dV7Ce3YLzoXH7" target="_blank">this form</a>.
+Podman Desktop is a graphical interface that enables seamless development with containers and Kubernetes.
```
Comment (slemeur) [inline README.md]: I think it's not accurate and precise enough.

Giving it a try:

Podman Desktop is a graphical interface that enables application developers to seamlessly work with containers and Kubernetes. 

Podman Desktop installs, configures and keeps Podman up to date on your local environment. It provides a system tray, to check status and interact with your container engine without losing focus from other tasks. The desktop application provides a dashboard to interact with containers, images, pods and volumes but also configure your environment with your OCI registries and network settings. Podman Desktop also provides capabilities to connect and deploy pods to Kubernetes environments. 

Podman Desktop also support multiple container engines, pick your favourite one and use the tool!

File: README.md [hunk 4]
```diff
@@ -15,70 +15,74 @@ Desktop App to interact with containers and pods running in Podman. Connect and
...
-- Docker
+We are seeking developers who are interested in improving Podman Desktop!
 
-## Download
+Register your interest to join our Early Adopter Program by filling <a href="https://forms.gle/ow73dV7Ce3YLzoXH7" target="_blank">this form</a>.
+
+This is totally *optional* and does not change any Podman Desktop features.
 
-Find latest binaries for Windows, MacOS and Linux on https://github.com/containers/podman-desktop/releases/
+## Download
```
Comment (slemeur) [inline README.md]: maybe we could include a link to the download page earlier in the readme

**Comment by benoitf** [comment]
@slemeur PTAL there were some commits after your review

**Review by slemeur — APPROVED** [review]
LGTM

**Pull request merged**